src/server.c

```diff
@@ -80,7 +81,11 @@ int dqlite__init(struct dqlite_node *d,
 			 "config__init(rv:%d)", rv);
 		goto err;
 	}
+	d->config.disk = disk_mode;
 	rv = VfsInit(&d->vfs, d->config.name);
+	if (disk_mode) {
+		dqlite_vfs_enable_disk(&d->vfs);
```

Comment:
We are not checking the return value of the function here.

Reply:
Yeah, good point.